components/tools/OmeroPy/test/integration/test_ishare.py

```diff
@@ -23,6 +23,25 @@
 
 class TestIShare(ITest):
 
+    @classmethod
+    def create_share(cls, description="", timeout=None,
```

Comment:
if share is deprecated now, how about if we add warning to test helpers as well?